python/cuspatial/cuspatial/core/gis.py

```diff
@@ -163,16 +172,36 @@ def point_in_polygon_bitmap(
     DataFrame: a DataFrame of Boolean values indicating whether each point
     falls within each polygon.
     """
-    bitmap_result = cpp_point_in_polygon_bitmap(
-        x_points, y_points, polygon_fpos, polygon_rpos, polygons_x, polygons_y
+
+    if len(poly_offsets) == 0:
+        return DataFrame()
+
```

Comment:
fwiw, the reason we're doing this is because: https://github.com/rapidsai/cudf/pull/5162